aptos-move/move-examples/tests/move_unit_tests.rs

```diff
@@ -46,12 +46,3 @@ fn test_hello_blockchain() {
     )]);
     run_tests_for_pkg("hello_blockchain", named_address);
 }
-
-#[test]
-fn test_message_board() {
```

Comment:
Why delete this? I think the MessageBoard module still exists right?

Reply:
I thought this is an example we have in tutorial, so we probably want to keep it